config.go

```diff
@@ -1442,20 +1406,20 @@ func ValidateConfig(cfg Config, interceptor signal.Interceptor, fileParser,
 		return nil, mkErr("log writer missing in config")
 	}
 
-	// Special show command to list supported subsystems and exit.
-	if cfg.DebugLevel == "show" {
-		fmt.Println("Supported subsystems",
-			cfg.LogWriter.SupportedSubsystems())
-		os.Exit(0)
-	}
-
 	if !build.SuportedLogCompressor(cfg.LogCompressor) {
 		return nil, mkErr("invalid log compressor: %v",
 			cfg.LogCompressor)
 	}
 
 	// Initialize logging at the default logging level.
 	SetupLoggers(cfg.LogWriter, interceptor)
+
```

Comment:
This is the fix, we need to first SetupLoggers, before we can print all the available ones.